easybuild/easyconfigs/m/MedPy/MedPy-0.4.0-foss-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,34 @@
+# This easyconfig was created by the BEAR Software team at the University of Birmingham.
+easyblock = 'PythonBundle'
+
+name = 'MedPy'
+version = '0.4.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = "https://pypi.org/project/MedPy/"
+description = """MedPy is a library and script collection for medical image processing in Python, providing basic
+ functionalities for reading, writing and manipulating large images of arbitrary dimensionality. Its main
+ contributions are n-dimensional versions of popular image filters, a collection of image feature extractors, ready
+ to be used with scikit-learn, and an exhaustive n-dimensional graph-cut package."""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+dependencies = [
+    ('Python', '3.7.4'),
+    ("SimpleITK", "1.2.4", versionsuffix),
+]
+
+use_pip = True
+
+exts_default_options = {
+    'source_urls': [PYPI_SOURCE],
+    'sanity_pip_check': True,
+}
+
+exts_list = [
+    (name, version, {
```

Comment:
@orbsmiv If there's only a single extension, you might as well use `PythonPackage` (and then not use `exts_list`)?

Reply:
Fixed in 2de401f